src/backend/distributed/sql/downgrades/citus--11.2-1--11.1-1.sql

```diff
@@ -0,0 +1 @@
+-- citus--11.0-1--10.2-3
```

Comment:
stale comment

Reply:
Looking into https://github.com/citusdata/citus/pull/5911, I feel like it would be better if we add similar sql files (upgrade / downgrade) for citus_columnar as well.

But besides that, I'm wondering what are the conditions that could make citus & citus_columnar extensions incompatible. I guess we should be able to deduce that information from CheckCitusColumnarVersion(). Or might ask to @yxu2162 ?